### ISSUE TITLE
Cannot install onto Nexus 7 deb

### ISSUE THREAD

**Issue by doniks** [ubports/ubuntu-touch#64]
I followed the instructions here:

https://devices.ubports.com/#/deb

However, it seems to fail with "can't flash recovery image". Pasted output below.

-----

Flashing stable channel

Creating filesystem with parameters:
    Size: 587202560
    Block size: 4096
    Blocks per group: 32768
    Inodes per group: 7168
    Inode size: 256
    Journal blocks: 2240
    Label: 
    Blocks: 143360
    Block groups: 5
    Reserved block group size: 39
Created filesystem with 11/35840 inodes and 4616/143360 blocks
erasing 'cache'...
OKAY [  0.631s]
sending 'cache' (10984 KB)...
OKAY [  0.362s]
writing 'cache'...
OKAY [  0.490s]
finished. total time: 1.483s
Creating filesystem with parameters:
    Size: 28856791040
    Block size: 4096
    Blocks per group: 32768
    Inodes per group: 8192
    Inode size: 256
    Journal blocks: 32768
    Label: 
    Blocks: 7045115
    Block groups: 215
    Reserved block group size: 1024
Created filesystem with 11/1761280 inodes and 154578/7045115 blocks
erasing 'userdata'...
OKAY [ 36.849s]
sending 'userdata' (139085 KB)...
OKAY [  4.475s]
writing 'userdata'...
OKAY [  5.999s]
finished. total time: 47.323s
rebooting into bootloader...
OKAY [  0.005s]
finished. total time: 0.055s

[sudo] password for peter: 
2017/05/29 22:03:58 Device is |deb|
2017/05/29 22:03:58 Flashing version 4 from ubuntu-touch/stable channel and server http://system-image.ubports.com to device deb
65.68 MB / 65.68 MB [================================================================================================================================] 100.00 % 1.52 MB/s 
312.91 MB / 312.91 MB [==============================================================================================================================] 100.00 % 2.89 MB/s 
can't flash recovery image

Move to your device to finish the setup.

Cleaning up..

Exiting script. Bye Bye

**Comment by NeoTheThird** [comment]
Sorry about that, but there is no working UBports build for the Nexus 7 yet. I updated the website.

**Comment by NeoTheThird** [comment]
I'll just reopen this to indicate that we're open for community developers to pick this device up.

**Comment by doniks** [comment]
I'm trying, but not getting very far yet: 
https://forums.ubports.com/topic/271/building-for-nexus-7-deb

**Comment by doniks** [comment]
Huh, I see that 

ubuntu-device-flash --verbose --server=https://system-image.ubports.com/ touch --device=deb --channel=ubuntu-touch/rc-proposed

seems to be installing and booting fine ... it's image version 6 .... so does that mean that someone IS working on this?!

**Comment by Flohack74** [comment]
No idea... @mariogrip what image was loaded into that channel, is this an old one?

**Comment by NeoTheThird** [comment]
@doniks I assume it's the old canonical version. Please don't put any valuable information in there, i'm assuming it's pretty old.

**Comment by NeoTheThird** [comment]
Duplicate of #643.
